projects/core/src/occ/adapters/configurator/variant/converters/occ-configurator-variant-normalizer.ts

```diff
@@ -49,12 +51,31 @@ export class OccConfiguratorVariantNormalizer
     attributeList.push(attribute);
   }
 
-  convertValue(occValue: OccConfigurator.Value, values: Value[]): void {
-    const value: Value = {
+  convertValue(
+    occValue: OccConfigurator.Value,
+    values: Configurator.Value[]
+  ): void {
+    const value: Configurator.Value = {
       valueCode: occValue.key,
       valueDisplay: occValue.langdepname,
     };
 
     values.push(value);
   }
+
+  mapOccConfiguratorVariantType(
```

Comment:
`OccConfiguratorVariantNormalizer` class name tells the context, so IMO no need to repeat it in the method's name `mapOccConfiguratorVariantType`. The `convertCharacteristicType` would be enough.

Reply:
done